src/components/BooksList.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+function BooksList() {
+  const books = useSelector((state) => state);
+  return (
+    <table>
+      <thead>
+        <tr>
+          <td>ID</td>
+          <td>Title</td>
+          <td>Category</td>
+        </tr>
+      </thead>
+      <tbody>
+        {books.map((book) => (
+          <tr key={book.id}>
+            <td>{book.id}</td>
+            <td>{book.title}</td>
+            <td>{book.category}</td>
+          </tr>
```

Comment:
Please use your `Book` component to render your books. This current code makes your `Book` component redundant. 
([screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/BooksList/BooksAsTable.png))